{
  "comments": [
    {
      "key": {
        "uuid": "e7cb4a83_4fdf75df",
        "filename": "/COMMIT_MSG",
        "patchSetId": 9
      },
      "lineNbr": 20,
      "author": {
        "id": 1365394
      },
      "writtenOn": "2020-03-11T13:33:14Z",
      "side": 1,
      "message": "Nit: Something went wrong here.\n\nAlso, can you attach `Bug: 1011811`?",
      "range": {
        "startLine": 20,
        "startChar": 0,
        "endLine": 20,
        "endChar": 52
      },
      "revId": "ea207d9ec85fe7b972e356ab991729c438f522e7",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    }
  ]
}